lib/util/template/index.ts

```diff
@@ -123,7 +123,8 @@ export const allowedFields = {
   sourceRepoOrg: 'The repository organization in the sourceUrl, if present',
   sourceRepoSlug: 'The slugified pathname of the sourceUrl, if present',
   sourceUrl: 'The source URL for the package',
-  updateType: 'One of digest, pin, rollback, patch, minor, major, replacement',
+  updateType:
+    'One of digest, pin, rollback, patch, minor, major, replacement, pinDigest',
```

Comment:
if we choose update type Digest, and rageStrategy=pin, 
doesnt that make it a pinDigest?